docs/Concepts/4.md

```diff
@@ -1,60 +1,104 @@
 ---
-title: Security Features
+title: Discovery & Bootstrap Nodes
 ---
 
-Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns.
-Therefore, the overall security of Waku v2 depends on how the different layers are utilized.
+Node discovery is the mechanism that enables a Waku node to find other nodes.
 
-The security models are detailed in the RFC of the protocols.
-That we strive to provide well documented and open source RFCs so that Waku users know what security guarantees are and aren't provided by each protocol.
+When starting a Waku node, it needs to connect to other nodes to be able to send, receive and retrieve messages.
+This means there needs to be a discovery mechanism that enables finding other nodes when its initially not connected to any nodes.
+This process is called _bootstrapping_.
 
-Some of the security features of Waku v2 are:
+Once connected, the node needs to find additional peers to have:
 
-### Pseudonymity
+- Enough peers in the Waku Relay mesh (target is 6),
+- Enough peers in reserve, in case current peers are overloaded or go offline,
+- Peers with specific Waku capabilities (e.g. Store, Light Push, Filter).
 
-Waku v2 by default guarantees pseudonymity for all of the protocol layers since parties do not have to disclose their true identity
-and instead they utilize libp2p `PeerID` as their identifiers.
-While pseudonymity is an appealing security feature, it does not guarantee full anonymity since the actions taken under the same pseudonym
-i.e., `PeerID` can be linked together and potentially result in the re-identification of the true actor.
+## Default Bootstrap Mechanism
 
-### Anonymity / Unlinkability
+The default bootstrap mechanism is to connect to the Status' nim-waku prod fleet.
 
-At a high level, anonymity is the inability of an adversary in linking an actor to its data/performed action (the actor and action are context-dependent).
-To be precise about linkability, we use the term Personally Identifiable Information (PII) to refer to any piece of data that could potentially be used to uniquely identify a party.
-For example, the signature verification key, and the hash of one's static IP address are unique for each user and hence count as PII.
-Notice that users' actions can be traced through their PIIs (e.g., signatures) and hence result in their re-identification risk.
-As such, we seek anonymity by avoiding linkability between actions and the actors / actors' PII. Concerning anonymity, Waku v2 provides the following features:
+You can find more information on how to configure your application/service to use this in the language-specific documentation: [js-waku](https://github.com/waku-org/js-waku), [nim-waku](https://github.com/status-im/nwaku) and [go-waku](https://github.com/waku-org/go-waku)
```

Comment:
```suggestion
```

There will/should be a global way/link to access implementaiton specific doc which is not finalized at this stage.
Introducing links now means more links to update soon so I'd refrain from now.